ais/models.py

```diff
@@ -852,6 +863,10 @@ class AddressSummary(db.Model):
     geocode_type = db.Column(db.Text)
     geocode_x = db.Column(db.Float)
     geocode_y = db.Column(db.Float)
+    # geocode_curb_x = db.Column(db.Float)
+    # geocode_curb_y = db.Column(db.Float)
+    geocode_street_x = db.Column(db.Float)
```

Comment:
can we take these out of address summary and query them from the geocode table as needed? reasons being:

1. the names are confusing to me. they're not street xy's, they're parcels that have been projected into the street. looking at this table i would think they're centerline xy's or true range, something like that.
2. there's nothing inherently special about the parcel_street geocodes. they're just another type. i think giving them their own field here is making a false distinction that, again, is going to get confusing.